test/org/sagebionetworks/bridge/TestUtils.java

```diff
@@ -55,11 +58,21 @@ public void run() {
         Http.RequestBody body = mock(Http.RequestBody.class);
         when(body.asJson()).thenReturn(node);
 
+        Map<String,String[]> headers = Maps.newHashMap();
+        headers.put("Content-Type", new String[] {"text/json; charset=UTF-8"});
         Http.Request request = mock(Http.Request.class);
+        Http.Response response = mock(Http.Response.class);
+
+        when(request.getHeader(anyString())).thenAnswer(invocation -> {
+            Object[] args = invocation.getArguments();
+            return headers.get(args[0])[0];
```

Comment:
Minor: Since we only ever use the first element anyway, could headers just be a Map<String, String>?